src/utils/daylight-utils.ts

```diff
@@ -63,14 +79,19 @@ export function getSolarNoonIntensity(dayNum: number, latitude: number): number
   const declination = 23.45 * Math.sin((360/365) * (dayNum - 81) * Math.PI / 180);
   const declinationRad = declination * Math.PI / 180;
   const dayAngle = 2 * Math.PI * (dayNum - 1) / 365;
-  // correction factor for Earth's elliptical orbit
-  const eccentricityFactor = 1 + 0.033 * Math.cos(dayAngle);
-  // cosine of the solar zenith angle at solar noon
+  const eccentricityFactor = 1 + 0.033 * Math.cos(dayAngle); // correction factor for Earth's elliptical orbit
   const cosSolarZenithAngle = Math.sin(latitudeRad) * Math.sin(declinationRad) +
                               Math.cos(latitudeRad) * Math.cos(declinationRad);
 
   const solarNoonIntensity = solarConstant * eccentricityFactor * cosSolarZenithAngle;
-  return Math.max(0, solarNoonIntensity); // Ensure non-negative value
+
+  // Note: This calculation returns theoretical intensity at the top of the atmosphere.
+  // Negative values are clamped to zero, which
+  // represents times when the sun is below the horizon
+  // In reality, some diffuse light might still be present due to atmospheric scattering.
+  // None of the calculations in this plugin use "civil twighlight" or associated definitions
```

Comment:
```suggestion
  // None of the calculations in this plugin use "civil twilight" or associated definitions
```
unless "twighlight" is a thing and I just haven't heard of it.